gradle/gradle-mvn-push.gradle

```
@@ -79,6 +79,23 @@ afterEvaluate {
                         developerConnection = POM_SCM_CONNECTION
                         url = POM_URL
                     }
+                    developers {
+                        developer {
+                            id = 'cortinico'
+                            name = 'Nicola Corti'
+                            email = 'corti.nico@gmail.com'
+                        }
+                        developer {
+                            id = 'vbuberen'
+                            name = 'Volodymyr Buberenko'
+                            email = 'vbuberenko@gmail.com'
```

Small typo in my email:
```suggestion
                            email = 'v.buberenko@gmail.com'
```